ruby-challenges.rb

```diff
@@ -3,32 +3,81 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
-reposts1 = 7
+# reposts1 = 7
 # Expected output: '7 is odd'
-reposts2 = 42
+# reposts2 = 42
 # Expected output: '42 is even'
-reposts3 = 221
+# reposts3 = 221
 # Expected output: '221 is odd'
 
+# def odd_number(reposts3)
+#     if reposts3 % 2 != 0
+#      "#{reposts3} is odd"
+# else 
+#     reposts3 %2 == 0
+#     "#{reposts3} is even"
+# end
+# end
+# p odd_number(reposts3)
+
+# output: 
+# "7 is odd"
+
+# output:
+# '42 is even'
+
+# output:
+# "221 is odd"
+
+
 
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. 
 # HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
-beatles_album1 = 'Rubber Soul'
+# "hello".delete "aeiou", "^e"
+
+# beatles_album1 = 'Rubber Soul'
+# p beatles_album1.delete "aeiou"
+# Output: "Rbbr Sl"
 # Expected output: 'Rbbr Sl'
-beatles_album2 = 'Sgt Pepper'
+
+
+# beatles_album2 = 'Sgt Pepper'
 # Expected output: 'Sgt Pppr'
-beatles_album3 = 'Abbey Road'
-# Expected output: 'bby Rd'
+# p beatles_album2.delete "aeiou"
+# Output: "Sgt Pppr"
 
 
+# beatles_album3 = 'Abbey Road'
+# Expected output: 'bby Rd'
+# p beatles_album3.delete "aeiou" "^A"
```

Comment:
The prompt is asking for a method, not just using built-in methods on each test variable